client/api/Repository.cs

```diff
@@ -64,6 +67,12 @@ public FeatureConfig GetFlag(string identifier)
         {
             return GetFlag(identifier, true);
         }
+        
+        public List<string> GetFlags()
```

Comment:
shouldnt this return an immutable/readonly clone instead?

Reply:
Agreed - thanks. Done.